oak-upgrade/src/main/java/org/apache/jackrabbit/oak/upgrade/cli/parser/StoreType.java

```diff
@@ -166,6 +168,7 @@ public StoreFactory createFactory(String[] paths, MigrationDirection direction,
                                 direction == MigrationDirection.SRC)
                         .connectionString(config.get(KEY_CONNECTION_STRING))
                         .containerName(config.get(KEY_CONTAINER_NAME))
+                        .sasToken(config.get(KEY_SHARED_ACCESS_SIGNATURE))
```

Comment:
Setting the SAS token here is not strictly necessary because it's by definition already part of the connection string (given that it's extracted from the connection string by the call to `parseAzureConfigurationFromCustomConnection(path)` just above).
However, in the `else` branch below, if a SAS token is part of the URL parsed by `parseAzureConfigurationFromUri(path)` (where URL has form `https://path/container?sasToken`) then it should be set in the builder, otherwise the `AZURE_SECRET_KEY` env var will be used in that case.